doubly_linked_list/doubly_linked_list.py

```diff
@@ -49,24 +58,48 @@ def add_to_head(self, value):
       # set head and tail to new listnode
       self.head = new_node
       self.tail = new_node
-  
-
 
   def remove_from_head(self):
-    pass
+    new_head = self.head
+    removed_head = None
+    if self.head:
+      new_head = self.head.next
+      if new_head:
+        print('here', new_head.value)
+        removed_head = self.head.value
+        return removed_head
+      self.tail = None
+    return None
 
   def add_to_tail(self, value):
```

Comment:
I have pinpointed your issue to your add_to_tail function.

 In the listNode class the insert_after method that may be helpful here.